packages/language-picker/src/language-picker.tsx

```diff
@@ -0,0 +1,115 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export type LanguageGroup = {
+	id: string;
+	name: () => string;
+	subTerritories?: string[];
+	countries?: string[];
+	default?: boolean;
+};
+
+type LanguageSlug = string;
+type WPLocale = string;
+
+type BaseLanguage = {
+	langSlug: LanguageSlug;
+	name: string;
+	popular?: number;
+	rtl?: boolean;
+	territories: string[];
+	value: number;
+	wpLocale: WPLocale | '';
+};
+
+type SubLanguage = BaseLanguage & { parentLangSlug: string };
+
+export type Language = BaseLanguage | SubLanguage;
+
+type Props = {
+	onSelectLanguage: ( language: Language ) => void;
+	languages: Language[];
+	languageGroups: LanguageGroup[];
+	defaultLananguageGroupId: string;
```

Comment:
There's a typo here: `defaultLananguageGroupId` -> `defaultLanguageGroupId`.

```suggestion
	defaultLanguageGroupId: string;
```